src/sentry/utils/performance_issues/detectors/consecutive_http_detector.py

```diff
@@ -136,4 +136,4 @@ def is_creation_allowed_for_organization(self, organization: Organization) -> bo
         )
 
     def is_creation_allowed_for_project(self, project: Project) -> bool:
-        return False
+        return True
```

Comment:
Kinda confused why this is in the diff.
https://github.com/getsentry/sentry/pull/45780/files